### PR TITLE
Add update and delete queries

### PR DESCRIPTION
**Pull request by farouqdaif1**
Add a column species of type string to your animals table. 
Insert the following data:
Animal: His name is Charmander. He was born on Feb 8th, 2020, and currently weighs -11kg. He is not neutered and he has never tried to escape.
Animal: Her name is Plantmon. She was born on Nov 15th, 2021, and currently weighs -5.7kg. She is neutered and she has tried to escape 2 times.
Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to escape 3 times.
Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
Animal: His name is Boarmon. He was born on Jun 7th, 2005, and currently weighs 20.4kg. He is neutered and he has tried to escape 7 times.
Animal: Her name is Blossom. She was born on Oct 13th, 1998, and currently weighs 17kg. She is neutered and she has tried to escape 3 times.
Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.
Inside a transaction update the animals table by setting the species column to unspecified. Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.
Inside a transaction:
Update the animals table by setting the species column to digimon for all animals that have a name ending in mon.
Update the animals table by setting the species column to pokemon for all animals that don't have species already set.
Commit the transaction.
Verify that change was made and persists after commit.
Now, take a deep breath and... Inside a transaction delete all records in the animals table, then roll back the transaction.
After the roll back verify if all records in the animals table still exist. After that you can start breath as usual ;)
Inside a transaction:
Delete all animals born after Jan 1st, 2022.
Create a savepoint for the transaction.
Update all animals' weight to be their weight multiplied by -1.
Rollback to the savepoint
Update all animals' weights that are negative to be their weight multiplied by -1.
Commit transaction
Write queries to answer the following questions:
How many animals are there?
How many animals have never tried to escape?
What is the average weight of animals?
Who escapes the most, neutered or not neutered animals?
What is the minimum and maximum weight of each type of animal?
What is the average number of escape attempts per animal type of those born between 1990 and 2000?

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -10,4 +10,43 @@ VALUES ('Pikachu','2021-1-7',1,false,15.04);
...
+--Animal: His name is Squirtle. He was born on Apr 2nd, 1993, and currently weighs -12.13kg. He was not neutered and he has tried to 3 times.
+
+insert into animals (name , date_of_birth,weight_kg,neutered,escape_attempts)
+values('Squirtle','1993-04-02',-12.13,false,3);
+
+--Animal: His name is Angemon. He was born on Jun 12th, 2005, and currently weighs -45kg. He is neutered and he has tried to escape once.
+
+
+insert into animals (name , date_of_birth,weight_kg,neutered,escape_attempts)
+values('Angemon','2005-06-12',-5.7,true,2);
```
Comment (SajjadAhmad14) [inline data.sql]: The value of `weight_kg` for `Angemon` should be `-45.0`.

File: queries.sql [hunk 2]
```diff
@@ -42,3 +42,122 @@ select * from animals where not name='Gabumon';
...
 --   3 | Pikachu | 2021-01-07    |               1 | f        |     15.04
 --   4 | Devimon | 2017-05-12    |               5 | t        |      11.0
+------------------------------------------------------------------Day 2-------------------------------
+-- Inside a transaction update the animals table by setting the species column to unspecified. 
+--Verify that change was made. Then roll back the change and verify that species columns went back to the state before transaction.
+
+begin;
+--BEGIN
+update animals set species ='unspecified';
+--UPDATE 11
```
Comment (SajjadAhmad14) [inline queries.sql:52]: Please verify that change was made before and after `rollback`

File: queries.sql [hunk 3]
```diff
@@ -42,3 +42,122 @@ select * from animals where not name='Gabumon';
...
+
+
+
+
+*/
+begin;
+update animals set species ='digimon' where name like '%mon';
+-- UPDATE 6
+update animals set species ='pokemon' where species is null ;
+--UPDATE 5
```
Comment (SajjadAhmad14) [inline queries.sql:75]: Please verify that change was made both before and after `commit`

File: data.sql [hunk 4]
```diff
@@ -10,4 +10,43 @@ VALUES ('Pikachu','2021-1-7',1,false,15.04);
 --INSERT 0 1
 insert into animals(name,date_of_birth,escape_attempts,neutered,weight_kg)
```
Comment (SajjadAhmad14) [inline data.sql:11]: Please add `project title` and `project description` to the `README` file. A professional README file should have complete details about the project.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Farouq, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
**_Note: Please do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AshakaE — APPROVED** [review]
Hi @farouqdaif1 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (farouqdaif1) [reply data.sql]: Thanks, I will do the changes

File: queries.sql (hunk 2 above)
Comment (farouqdaif1) [reply queries.sql:52]: Thanks, I will do the changes

File: queries.sql (hunk 3 above)
Comment (farouqdaif1) [reply queries.sql:75]: Thanks, I will do the changes

File: data.sql (hunk 4 above)
Comment (farouqdaif1) [reply data.sql:11]: Thanks, I will do the changes

**Pull request merged**